CreatubblesAPIClient/Sources/Model/Notification.swift

```diff
@@ -91,4 +91,13 @@ open class Notification: NSObject, Identifiable
         bubbleRelationship = MappingUtils.relationshipFromMapper(mapper.bubbleRelationship)
         bubble = MappingUtils.objectFromMapper(dataMapper, relationship: bubbleRelationship, type: Bubble.self)
     }
+    
+    var isValid: Bool
```

Comment:
that's fine then, but I do think that removing the force unwrap is an easy win here since it is not required in any way.